core/src/main/kotlin/org/sertain/command/Subsystem.kt

```diff
@@ -0,0 +1,11 @@
+@file:Suppress("unused", "RedundantVisibilityModifier")
+package org.sertain.command
+
+import org.sertain.RobotLifecycle
+
+private typealias WpiLibSubsystem = edu.wpi.first.wpilibj.command.Subsystem
+
+/**
+ * @see edu.wpi.first.wpilibj.command.Subsystem
+ */
+public abstract class Subsystem : WpiLibSubsystem(), RobotLifecycle
```

Comment:
In `onCreate()`, you gotta add a listener for the lifecycle.

Reply:
Why `onCreate()` and not `init` like in Command?

Reply:
I did `init` just to mimic Command for now. Will change it to `onCreate()` if that's how it should be done.

Reply:
Cuz this is a library and people will be extending the class. Example: someone adds a field that they use in one of the callbacks -> this callback is added in the super constructor before the field has been initialized -> NPE.